lib/rex/post/meterpreter/ui/console/command_dispatcher/extapi/clipboard.rb

```diff
@@ -365,52 +378,69 @@ def cmd_clipboard_monitor_stop(*args)
         download_images = val.downcase != 'false'
       when "-f"
         download_files = val.downcase != 'false'
+      when '--force'
+        force_overwrite = true
       when "-h"
         print_clipboard_monitor_stop_usage
         return true
       end
     }
+    
 
     dump = client.extapi.clipboard.monitor_stop({
       :dump           => dump_data,
       :include_images => download_images
     })
 
-    parse_dump(dump, download_images, download_files, download_path) if dump_data
+    parse_dump(dump, download_images, download_files, download_path, force_overwrite) if dump_data
 
     print_good("Clipboard monitor stopped")
   end
 
 private
 
-  def download_file( dest_folder, source )
+  def download_file( dest_folder, source, force_overwrite=false )
     stat = client.fs.file.stat( source )
     base = ::Rex::Post::Meterpreter::Extensions::Stdapi::Fs::File.basename( source )
+    attempted_overwrite = false
 
     # Basename ends up with a single name/folder. This is the only point where it
     # may be possible to do a dir trav up one folder. We need to check to make sure
     # that the basename doesn't result in a traversal
-    return false if base == '..'
+    return false,attempted_overwrite if base == '..'
 
     dest = File.join( dest_folder, base )
 
     if stat.directory?
+      if ::File.file?(dest) && !force_overwrite
+          print_error("Cannot write to existing files if force overwrite is not enabled")
+          attempted_overwrite = true
+      else
       client.fs.dir.download( dest, source, {"recursive" => true}, true ) { |step, src, dst|
```

Comment:
Would we be able to rename these `dst`, `dest` and `src` variables to make it a bit easier to understand what they represent?
Can ignore as they have been here before this PR👍